MovieFlix/app/build.gradle

```diff
@@ -0,0 +1,62 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'com.example.movieflix'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "com.example.movieflix"
+        minSdk 24
+        targetSdk 33
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+    buildFeatures {
+        dataBinding true
+
+    }
+}
+
+dependencies {
```

Comment:
it is a good practice to either sort all your dependencies either by the library they are importing,in that case comment library name above relevant dependencies, or in other cas simply sort it by the dependencies e.g. the dependencies starting with com should come after androidx dependencies, etc